detekt/defs.bzl

```diff
@@ -94,11 +94,10 @@ def _impl(ctx):
         action_outputs.append(html_report)
         detekt_arguments.add("--report", "html:{}".format(html_report.path))
 
-    if ctx.attr._txt_report:
-        txt_report = ctx.actions.declare_file("{}_detekt_report.txt".format(ctx.label.name))
+    txt_report = ctx.actions.declare_file("{}_detekt_report.txt".format(ctx.label.name))
```

Comment:
If the `_txt_report` ATTR isn't being used anymore then we should just remove it.